src/Text/Pandoc/Extensions.hs

```diff
@@ -118,6 +118,7 @@ data Extension =
     | Ext_literate_haskell    -- ^ Enable literate Haskell conventions
     | Ext_markdown_attribute      -- ^ Interpret text inside HTML as markdown iff
                                   --   container has attribute 'markdown'
+    | Ext_wikilink -- ^ Interpret a markdown wiki link
```

Comment:
Sorry, but on reflection I'm thinking `Ext_wikilinks` would be more consistent with the extension naming style.